package.json

```diff
@@ -2,7 +2,7 @@
   "name": "llnode",
   "version": "1.6.2",
   "description": "Node.js plugin for LLDB",
-  "main": "no-entry-sorry.js",
+  "main": "index.js",
```

Comment:
One problem: what should `npm install` build, the addon or the plugin?

Reply:
Agree, it would be great if `npm install llnode` built both the addon/API and the lldb plugin.